tests/requirements.txt

```diff
@@ -1,2 +1,2 @@
-jsonref
-jsonschema
+jsonschema>=3.0.0a3
```

Comment:
It looks like `jsonschema` finally has [v3 production releases](https://pypi.org/project/jsonschema/#history). Hence, I would change this line to `jsonschema>=3.0`.

Reply:
thanks @bdarcus for addressing in https://github.com/citation-style-language/schema/pull/168/commits/26cccd0e67759a073af7a9e401b1c5cdf9025855. This shouldn't actually change anything functionally but looks a bit cleaner.